WordPress/Classes/Services/CommentService.m

```diff
@@ -14,6 +14,7 @@
 NSUInteger const WPTopLevelHierarchicalCommentsPerPage = 20;
 NSInteger const  WPNumberOfCommentsToSync = 100;
 static NSTimeInterval const CommentsRefreshTimeoutInSeconds = 60 * 5; // 5 minutes
+NSString *commentStatusAll = @"all";
```

Comment:
I need to define all the endpoint `status` types, but I'm not sure yet where/how that will be. This will do for now since we're just using `all`.